src/layouts/EditPage.jsx

```diff
@@ -51,8 +53,17 @@ const EditPage = ({ match }) => {
   const [hasChanges, setHasChanges] = useState(false)
   const [isContentViolation, setIsContentViolation] = useState(false)
   const [isXSSViolation, setIsXSSViolation] = useState(false)
-  const [showXSSWarning, setShowXSSWarning] = useState(false)
-  const [showOverwriteWarning, setShowOverwriteWarning] = useState(false)
+
+  const {
+    isOpen: isXSSOpen,
+    onOpen: onXSSOpen,
+    onClose: onXSSClose,
+  } = useDisclosure()
+  const {
+    isOpen: isOverwriteOpen,
+    onOpen: onOverwriteOpen,
+    onClose: onOverwriteClose,
+  } = useDisclosure()
```

Comment:
nit: could we give more descriptive names to these props from `useDisclosure`?

Reply:
Whoops missed this out, fixed in https://github.com/isomerpages/isomercms-frontend/pull/955/commits/0b96057bf0edf41867940922cae721ef656958d2